### ISSUE TITLE
Console: print values from single function calls

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#99]
when an input is entered that could be interpreted as a single function call expression, its output should be printed, instead of printing nothing (this would be in line with the lua 5.3 repl but not the 5.1 one)

**Comment by HeladoDeBrownie** [comment]
resolved by https://github.com/HeladoDeBrownie/Nexus/commit/2782f583daa3ceaad83201ba49fc3fe6b7e1bc15, which also allows general expression lists much like the lua 5.3 repl does
